rel-eng/releasers.conf

```diff
@@ -1,3 +1,3 @@
 [koji-katello-pulpcore]
 releaser = tito.release.KojiReleaser
-autobuild_tags = pulpcore-3.18-el9 pulpcore-3.18-el8
+autobuild_tags = pulpcore-3.17-el7 pulpcore-3.18-el8 pulpcore-3.18-el9
```

Comment:
er, this should probably be pulpcore-3.18-el7

Reply:
My bad, fixed